src/Host/Client/Test/Session/RSessionTest.InteractionEvaluation.cs

```diff
@@ -79,6 +79,42 @@ public class InteractionEvaluation : IAsyncLifetime {
                     completedTasks.Single().Result.Dispose();
                 }
             }
+
+            [Test]
+            [Category.R.Session]
+            public async Task NestedInteraction() {
```

Comment:
Where should that go? At the beginning of the task method? Do I need to do anything at the end to clean it up?